src/vcf/vcf_v41.ragel

```diff
@@ -74,36 +74,36 @@
     ## - The line is a single key-value pair (meta_value): the value cannot start with an angle bracket, quote or linebreak, but it is free text thereafter
     ## - One of multiple key-value pairs, non-quoted (meta_field_value): the value cannot contain a comma -end of field-, or a closing angle bracket -end of entry-, or quote
     ## - One of multiple key-value pairs, quoted (meta_field_desc): the value can contain any character except for non-escaped quotes
-    meta_value      = ((print - ("<" | "\"" | NL)) (print)*)	>token_begin $token_middle %token_end ;
-    meta_field_value= (print - ("," | ">" | "\""))+		>token_begin $token_middle %token_end ;
-    meta_field_desc = (("\\" print) | (print - "\""))*		>token_begin $token_middle %token_end ;
+    meta_value      = ((print - ("<" | '"' | NL)) (print)*)	>token_begin $token_middle %token_end ;
```

Comment:
All the strings should be surrounded by single quotes, not only those that require escaping. Having some with single and some with double quotes will make the machine definition look inconsistent.